src/Native/Unix/System.Security.Cryptography.Native/openssl.h

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+
+#pragma once
+
+#include "pal_compiler.h"
+#include "opensslshim.h"
+
+extern "C" char* CryptoNative_SSLEayVersion();
```

Comment:
I have reduced the number of delcarations in this file to the bare minimum that's required to build.